src/v2/BaseLoginRouter.js

```diff
@@ -15,33 +15,26 @@
 // responsible for adding new routes)
 import { _, $, Backbone, Router, loc } from 'okta';
 import Settings from 'models/Settings';
-import BrowserFeatures from 'util/BrowserFeatures';
 import Bundles from 'util/Bundles';
 import ColorsUtil from 'util/ColorsUtil';
 import Enums from 'util/Enums';
 import Errors from 'util/Errors';
 import Logger from 'util/Logger';
+import LanguageUtil from 'util/LanguageUtil';
 import AuthContainer from 'views/shared/AuthContainer';
 import Header from 'views/shared/Header';
 import responseTransformer from './ion/responseTransformer';
 import uiSchemaTransformer from './ion/uiSchemaTransformer';
 import i18nTransformer from './ion/i18nTransformer';
 import AppState from './models/AppState';
+import idx from 'idx';
 
-function loadLanguage (appState, languageCode, i18n, assetBaseUrl, assetRewrite) {
-  const timeout = setTimeout(function () {
-    // Trigger a spinner if we're waiting on a request for a new language.
-    appState.trigger('loading', true);
-  }, 200);
-
-  return Bundles.loadLanguage(languageCode, i18n, {
-    baseUrl: assetBaseUrl,
-    rewrite: assetRewrite,
-  }).then(function () {
-    clearTimeout(timeout);
-    appState.trigger('loading', false);
-  });
-}
+const introspectStateToken = (settings) => {
```

Comment:
Is https://github.com/okta/okta-signin-widget/blob/bacon-topic-display-introspect-error/src/util/Util.js#L219 being called anymore ? Can we remove it ?

Reply:
yes, it can be removed.